docs/examples/axe-earl-report-example.json

```diff
@@ -0,0 +1,248 @@
+{
```

Comment:
- [ ] This needs to shrink to like 10 usable assertions.
- [ ] Change subject urls to match with example v3 data sample